services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/models/EmailSettings.java

```diff
@@ -0,0 +1,120 @@
+package us.dot.its.jpo.ode.api.models;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import lombok.extern.slf4j.Slf4j;
+import us.dot.its.jpo.geojsonconverter.DateJsonMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Slf4j
+public class EmailSettings {
+    public boolean receiveAnnouncements;
+    public boolean receiveCeaseBroadcastRecommendations;
+    public boolean receiveCriticalErrorMessages;
+    public boolean receiveNewUserRequests;
+    public EmailFrequency notificationFrequency;
```

Comment:
chore(blocking): properly set member variable access levels to private
```suggestion
    private boolean receiveAnnouncements;
    private boolean receiveCeaseBroadcastRecommendations;
    private boolean receiveCriticalErrorMessages;
    private boolean receiveNewUserRequests;
    private EmailFrequency notificationFrequency;
```